specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2020-06-01-preview/sqlX.json

```diff
@@ -0,0 +1,261 @@
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/accountNameParameter"
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "The sqlX delete operation will complete asynchronously."
+          },
+          "204": {
+            "description": "SqlX service does not exist in the database account."
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "cosmos-db.json#/definitions/DefaultErrorResponse"
+            }
+          },
+          "x-ms-examples": {
+            "CosmosDBServiceResourceDelete": {
+              "$ref": "./examples/CosmosDBSqlAcceleratorDelete.json"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "ServiceResourceCreateUpdateParameters": {
+      "description": "Parameters to create a service resource",
+      "type": "object",
+      "properties": {
+        "properties": {
+          "description": "Properties to create and update an Azure Cosmos DB Service Resource.",
+          "$ref": "#/definitions/ServiceResource",
+          "x-ms-client-flatten": true
+        }
+      },
+      "required": [
+        "properties"
+      ]
+    },
+    "ServiceGetResults": {
+      "description": "An Azure Cosmos DB Cassandra table.",
+      "type": "object",
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "description": "The properties of an Azure Cosmos DB Cassandra table",
+          "$ref": "#/definitions/ServiceResource"
+        }
+      }
+    },
+    "ServiceResource": {
+      "description": "A CosmosDB Service resource",
+      "type": "object",
+      "properties": {
+        "serviceType": {
+          "description": "Specifies the type of Service.",
+          "type": "string",
+          "enum": [
+            "Sqlx"
+          ]
+        },
+        "status": {
+          "description": "Status of the CosmosDB Service.",
+          "type": "string"
+        },
+        "instanceSize": {
+          "description": "Size of the CosmosDB Service.",
+          "type": "string"
+        },
+        "instanceCount": {
+          "description": "Instance count of the CosmosDB Service.",
+          "type": "string"
+        },
+        "locations": {
+          "description": "Map of a location name to regional service resource.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/RegionalServiceResource"
+          }
+        }
+      }
+    },
+    "RegionalServiceResource": {
+      "description": "Properties of a regional service resource.",
+      "type": "object",
+      "properties": {
+        "name": {
```

Comment:
I will create a PR to remove the name property. Can we remove it from here as well?

Reply:
sure thing